benchmarkapps/Sample/Startup.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Sample
+{
+    public class Startup
+    {
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddRouting();
+        }
+
+        public void Configure(IApplicationBuilder app)
+        {
+            app.UseRouter(routes =>
+            {
+                routes.MapRoute("/", (httpContext) =>
```

Comment:
do you think this should match the things that the plaintext middleware does?
https://github.com/aspnet/benchmarks/blob/dev/src/Benchmarks/Middleware/PlaintextMiddleware.cs#L27

at the least I think it should need to match `/plaintext` as well as setting a content type.

Optionally, we can pre-encode the bytes.


I think all of this would be able to support our goal of understanding the cost of adding routing/dispatcher, and allow for an apples-to-apples comparison with the plaintext middleware.

Reply:
In this case just follow what the middleware plaintext scenario is doing, so it becomes a baseline for this new scenario. https://github.com/aspnet/benchmarks/blob/dev/src/Benchmarks/Middleware/PlaintextMiddleware.cs#L35